docs/api/server/CLI-Commands.md

```diff
@@ -0,0 +1,204 @@
+[👈 Return to Overview](../API.md)
+
+# CLI Commands
+
+`one-app` Server consists of [NPM Script Commands](https://docs.npmjs.com/misc/scripts) that determine its runtime configuration and behaviors. 
+
+```sh
+npm run <one-app-npm-command>
+# e.g. npm run test:unit -- --watch
+```
+
+We describe each command and a description of its usage below.
+
+**Contents**
+* Install Commands
+  * [`preinstall`](#install-commands)
+  * [`postinstall`](#install-commands)
+* Clean Commands
+  * [`clean`](#clean-commands)
+  * [`clean:build`](#clean-commands)
+  * [`clean:test`](#clean-commands)
+* Build Commands
+  * [`build`](#build-commands)
+  * [`build:server`](#build-commands)
+  * [`build:bundle`](#build-commands)
+  * [`build:artifacts:statics`](#build-commands)
+  * [`build:prod-sample`](#build-commands)
+  * [`build:sample-modules`](#build-commands)
+* Start Commands
+  * [`start`](#start-commands)
+  * [`start:watch`](#start-commands)
+* Test Commands
+  * [`test`](#test-commands)
+  * [`test:unit`](#test-commands)
+  * [`test:lint`](#test-commands)
+  * [`pretest:integration`](#test-commands)
+  * [`test:integration`](#test-commands)
+  * [`test:danger`](#test-commands)
+  * [`test:git-history`](#test-commands)
+  * [`test:lockfile`](#test-commands)
+  * [`prestart:prod-sample`](#test-commands)
+  * [`start:prod-sample`](#test-commands)
```

Comment:
do these belong under "Start commands"?

Reply:
Depends on how you see this. Im open to moving `start:prod-sample` to Start Commands.

Reply:
To me, it makes more sense under start commands

Reply:
Fixed in a88378aafc426e868db005159f4e769323b6a93d